dali/test/example_test.cc

```diff
@@ -0,0 +1,31 @@
+#include "dali_operator_test.h"
+
+namespace dali {
+
+namespace testing {
+
+class MyOperatorTest : public DaliOperatorTest<GPUBackend, CPUBackend> {
+ protected:
+  OperatorsGraph GenerateOperatorsGraph() const noexcept override {
+    return "MyOp";
+  };
+
+  std::vector<Input> inputs;
+  std::vector<Output> outputs;
+};
+
+TEST_F(MyOperatorTest, DISABLED_MyOperatorTestCase) {
```

Comment:
can you explain the purpose of this test?

Reply:
Outdated